src/main/java/com/google/ehub/servlets/ProfileServlet.java

```diff
@@ -27,67 +21,42 @@
 /** Servlet that returns HTML that contains the user's profile information.**/
 @WebServlet("/profile-data")
 public class ProfileServlet extends HttpServlet {
-<<<<<<< HEAD
-
   ProfileDatastore profileData = new ProfileDatastore();
   UserProfile newUser;
+
   @Override
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    
     ProfileDatastore data = new ProfileDatastore();
     String name = request.getParameter("name");
```

Comment:
You should probably make it so the parameter keys are stored in private static final String variables like this:
```
private static final String NAME_PARAMETER_KEY = "name";
```

Reply:
I think I can do that for certain keys because the value must change if a user decided to edit/update their profile

Reply:
The user should not be able to change the parameter keys of the Post request, instead they are only able to change the values associated with this keys.

Reply:
I don't think I understand the purpose of having it in the form you suggested. I use the format (private static final String NAME_PARAMETER_KEY = "name";) In my datastore file

Reply:
You can reuse those constants here. Its better to store hardcoded strings in constants, especially if they are used in multiple places. In the future, if you need to change the string, you only have to update it in one location.